### PR TITLE
feat(web): support property placeholder on infobox blocks

### PR DESCRIPTION
**Pull request by airslice**
# Overview

Added the support for fetch/process/display the placeholder on builtin infobox block editors.

## What I've done

- Update gql & process function
- Add placeholder text & translation to
  - Infobox Text block
  - Infobox Image block
  - Infobox Video block
  - Infobox Markdown block
  - Infobox PropertyList block

NOTE: PropertyList manage its own editor panel, the text is defined on FE.

## Release Notes

- **New Features**
  - Introduced placeholder attributes in various infobox components, enhancing user guidance for input fields.
  - Added a display type option for property lists, allowing users to customize the display of properties.

- **Bug Fixes**
  - Improved handling of new list items in the property list editor for better user experience.

- **Documentation**
  - Updated translations for English and Japanese, adding new phrases and improving existing ones for better clarity and usability.

These changes collectively improve the user interface and experience, providing clearer instructions and more customizable options.

### CODE REVIEW

**Review by pyshx — APPROVED** [review]

File: server/pkg/builtin/manifest_ja.yml [hunk 1]
```diff
@@ -1335,6 +1335,7 @@ extensions:
         fields:
           text:
             title: コンテンツ
+            placeholder: ${あなたのプロパティ値}
```
Comment (asahina820) [inline server/pkg/builtin/manifest_ja.yml:1338]: Does this mean "your property value" ?
If so, "プロパティの値" is better.

**Review by asahina820 — COMMENTED** [review]
I checked Japanese translation :)

File: server/pkg/builtin/manifest_ja.yml (hunk 1 above)
Comment (airslice) [reply server/pkg/builtin/manifest_ja.yml]: Yes! Updated, Thank you!

File: web/src/beta/features/Visualizer/Crust/Infobox/Block/builtin/PropertyList/ListEditor/Item.tsx [hunk 2]
```diff
@@ -44,18 +45,22 @@ const EditorItem: FC<Props> = ({
...
 
+  const t = useT();
+
   return (
     <Field>
       <HandleIcon icon="dotsSixVertical" className={handleClassName} />
       <TextInput
         size="small"
         value={currentKeyValue}
+        placeholder={t("Display title")}
```
[the remark this answers is not in the document]
Comment (airslice) [reply web/src/beta/features/Visualizer/Crust/Infobox/Block/builtin/PropertyList/ListEditor/Item.tsx:56]: No it won't i think.

**Review by mkumbobeaty — APPROVED** [review]

**Pull request merged**